index.html

```diff
@@ -37,6 +37,177 @@ <h1 class="intro">I’m a software developer</h1>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul> 
+                    <div class="project">
+                        <button>See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="work-item">
+                <div class="snapshot">
+                </div>
+                <div class="details">
+                    <h2>Multi-Post Stories Gain+Glory</h2>
+                    <ul class="tech-stack">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <div class="project">
+                        <button>See Project</button>
+                    </div> 
+                </div>
+            </div>
+
+            <div class="work-item">
+                <div class="snapshot">
+                </div>
+                <div class="details">
+                    <h2>Multi-Post Stories Gain+Glory</h2>
+                    <ul class="tech-stack">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul> 
+                    <div class="project">
+                        <button>See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="work-item">
+                <div class="snapshot">
+                </div>
+                <div class="details">
+                    <h2>Multi-Post Stories Gain+Glory</h2>
+                    <ul class="tech-stack">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul> 
+                    <div class="project">
+                        <button>See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+      
+    </section>
+    <section class="about_section">
+        <div class="about-details">
+            <h1>About me</h1>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+            </p>
+            <button >Get my resume</button>
+        </div>
+        
+        <div class="about-tech">
+            <div class="languages">
+                <div class="icon-language">
+                    <img src="images/about/icon-languages.png" alt="icon-languages">
+                </div>
+                <h2>Languages</h2>
+                <div class="list">
```

Comment:
- [ ] For semantic reasons, Kindly make sure your technologies categories("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
```html
<ul>
<li>Languages</li>
...
<li>Framewors</li>
...
</ul>
```